types/index.d.ts

```diff
@@ -0,0 +1,377 @@
+// TypeScript Version: 2.4
+export default Long;
+
+declare class Long {
+    /**
+     * Constructs a 64 bit two's-complement integer, given its low and high 32 bit values as signed integers. See the from* functions below for more convenient ways of constructing Longs.
+     */
+    constructor(low: number, high?: number, unsigned?: boolean);
+
+    /**
+     * Maximum unsigned value.
+     */
+    static MAX_UNSIGNED_VALUE: Long;
+
+    /**
+     * Maximum signed value.
+     */
+    static MAX_VALUE: Long;
+
+    /**
+     * Minimum signed value.
+     */
+    static MIN_VALUE: Long;
+
+    /**
+     * Signed negative one.
+     */
+    static NEG_ONE: Long;
+
+    /**
+     * Signed one.
+     */
+    static ONE: Long;
+
+    /**
+     * Unsigned one.
+     */
+    static UONE: Long;
+
+    /**
+     * Unsigned zero.
+     */
+    static UZERO: Long;
+
+    /**
+     * Signed zero
+     */
+    static ZERO: Long;
+
+    /**
+     * The high 32 bits as a signed value.
+     */
+    high: number;
+
+    /**
+     * The low 32 bits as a signed value.
+     */
+    low: number;
+
+    /**
+     * Whether unsigned or not.
+     */
+    unsigned: boolean;
+
+    /**
+     * Returns a Long representing the 64 bit integer that comes by concatenating the given low and high bits. Each is assumed to use 32 bits.
+     */
+    static fromBits(lowBits: number, highBits: number, unsigned?: boolean): Long;
+
+    /**
+     * Returns a Long representing the given 32 bit integer value.
+     */
+    static fromInt(value: number, unsigned?: boolean): Long;
+
+    /**
+     * Returns a Long representing the given value, provided that it is a finite number. Otherwise, zero is returned.
+     */
+    static fromNumber(value: number, unsigned?: boolean): Long;
+
+    /**
+     * Returns a Long representation of the given string, written using the specified radix.
+     */
+    static fromString(str: string, unsigned?: boolean | number, radix?: number): Long;
+
+    /**
+     * Creates a Long from its byte representation.
+     */
+    static fromBytes(bytes: number[], unsigned?: boolean, le?: boolean): Long;
+
+    /**
+     * Creates a Long from its little endian byte representation.
+     */
+    static fromBytesLE(bytes: number[], unsigned?: boolean): Long;
+
+    /**
+     * Creates a Long from its little endian byte representation.
+     */
+    static fromBytesBE(bytes: number[], unsigned?: boolean): Long;
+
+    /**
+     * Tests if the specified object is a Long.
+     */
+    static isLong(obj: any): boolean;
+
+    /**
+     * Converts the specified value to a Long.
+     */
+    static fromValue(val: Long | number | string | {low: number, high: number, unsigned: boolean}): Long;
```

Comment:
This function accepts a second argument `unsigned`, see: https://github.com/dcodeIO/long.js/blob/master/src/long.js#L280